scripts/generate-conda-envs.py

```diff
@@ -485,7 +488,9 @@ def section_selected(section):
             use=config.use,
             python=config.python,
             conda_sections=conda_sections,
-            pip=PIP_TEMPLATE.format(pip_sections=pip_sections),
+            pip=PIP_TEMPLATE.format(pip_sections=pip_sections)
+            if pip_sections
+            else "",
```

Comment:
I think it would be better to handle this in the template, to keep all the cases here consistent

Reply:
Thank you for the suggestion, reverted the change.